packages/react/src/constants/index.js

```diff
@@ -65,3 +59,29 @@ export const LIST_ITEM_VALUES_TYPES = {
   ERROR: "error",
   SUCCESS: "success",
 };
+
+export const UI_TYPES = {
+  Heading1: "Heading1",
+  Heading2: "Heading2",
+  Heading3: "Heading3",
+  Heading4: "Heading4",
+  Heading5: "Heading5",
+  Heading6: "Heading6",
```

Comment:
Just a suggestion: would be nice to have these as `H<X>` rather than `Heading<X>`

Reply:
Since we switched to `Heading-<X>`, it could be weird to have `H-1` don't you think?

Reply:
Nevermind 🤦‍♂,
Done!